Rubberduck.Deployment/Rubberduck.Deployment.csproj

```diff
@@ -23,6 +23,14 @@
     <Content Include="..\packages\EasyHook.2.7.6684\content\net40\**" CopyToPublishDirectory="PreserveNewest" Link="%(Filename)%(Extension)" />
     <Content Include="$(USERPROFILE)\.nuget\packages\easyHook\2.7.6684\content\net40\**" CopyToPublishDirectory="PreserveNewest" Link="%(Filename)%(Extension)" />
   </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="InnoSetup\Graphics\Ducky.ico" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="InnoSetup\Graphics\Ducky.ico" />
+  </ItemGroup>
```

Comment:
Does that make sense? Should we `None Remove` then `Content Include`?

Reply:
@bclothier I tried finding the old `Ducky.ico` in the old .csproj and it wasn't there - so this is being removed (diff is outdated)